fuzz/api/main.go

```diff
@@ -1,6 +1,7 @@
-package csecmd
+package api
```

Comment:
need to update https://github.com/Azure/AgentBaker/blob/master/.clusterfuzzlite/build.sh#L3

Reply:
the doc there has info - 
https://github.com/Azure/AgentBaker/blob/master/.clusterfuzzlite/README.md
https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh

the package name/func name just needs to match